filebeat/Makefile

```diff
@@ -5,6 +5,8 @@ SYSTEM_TESTS?=true
 TEST_ENVIRONMENT?=true
 GOX_FLAGS=-arch="amd64 386 arm ppc64 ppc64le"
 ES_BEATS?=..
+FIELDS_FILE_PATH=module
```

Comment:
Does this only support one path? I could see especially in libbeat or filebeat that a module, processor and inputs have fields.yml.

Reply:
Yes, but it can be changed easily to support lists.